HackathonTestApp/app/src/main/java/com/sumologic/hackathontestapp/MainActivity.java

```diff
@@ -0,0 +1,152 @@
+package com.sumologic.hackathontestapp;
+
+import android.os.Bundle;
+import android.os.StrictMode;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.Snackbar;
+import android.view.View;
+import android.support.design.widget.NavigationView;
+import android.support.v4.view.GravityCompat;
+import android.support.v4.widget.DrawerLayout;
+import android.support.v7.app.ActionBarDrawerToggle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import java.io.IOException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import okhttp3.Headers;
+import okhttp3.OkHttpClient;
+import okhttp3.Response;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+
+
+public class MainActivity extends AppCompatActivity
+        implements NavigationView.OnNavigationItemSelectedListener {
+
+    private Logger log = LoggerFactory.getLogger(MainActivity.class);
+    private OkHttpClient client = new OkHttpClient();
+
+
+    private void sendLogData(String data) {
+        android.util.Log.e("Ashish",data);
+        log.info("hello world");
+        RequestBody body = RequestBody.create(null, "This is an example for the usage of OkHttp in a standard Java program");
```

Comment:
The second parameter for `RequestBody.create` should be `data`, right @ashishism10 ?